### ISSUE TITLE
Adding AI Primitives to Nostr

### ISSUE THREAD

**Issue by PeterLayer3** [nostrworld/nostriga#87]
# Adding AI primitives to Nostr
### Social Links
Nostr: npub1ses3rqgrcvx9kdu5hfnf4klgw2zhfs780cxcngmlf55s387z0tts267waq
and npub1r8vwncwpymx4mrvj4zgkrragnxulfazvfjnd80k9r0ahc55cgs6q0pkmca
Github:  https://github.com/source-c
Twitter: @peteryared
Website: layer3.news

# Presentation, demo, and interactive discussion
### We are there the whole time so whatever works if you want to include this!
*We will do our best to accommodate your requested time slot. Please let us know if there are any dates/times that ***absolutely do not work*** for you.*

**Comment by source-c** [comment]
There are also a slides here 
[Layer3.news-Nostriga.pdf](https://github.com/user-attachments/files/16665580/Layer3.news-Nostriga.pdf)

Also this topic is maybe somehow in overlapping with #52 (news media, censorship resistance, events analysis etc). Might be it is worth it to collaborate or join? This may become a fireside instead of keynotes, hm?

**Comment by exfrog** [comment]
I believe our Agenda is full, and submissions are closed, but there is a 4th stage with minimal av that can be used to present topics that didn't make it on other stages.

Later today when at a computer I'll accept this talk and place it in an open slot on the 4th stage. Feel free to reach out to the other panel to see if the topics are similar enough to merge efforts, but I expect they have their own plans already.

**Comment by source-c** [comment]
Yeah, I know this is late and goes like "last minute talk", however I believe @PeterLayer3 is a great speaker and definitely has what to say on this ([his site](https://yared.com)). Thank you for your effort anyway!

**Comment by PeterLayer3** [comment]
> I believe our Agenda is full, and submissions are closed, but there is a 4th stage with minimal av that can be used to present topics that didn't make it on other stages.
> 
> Later today when at a computer I'll accept this talk and place it in an open slot on the 4th stage. Feel free to reach out to the other panel to see if the topics are similar enough to merge efforts, but I expect they have their own plans already.

Thank you. Sorry for the late submission we got caught up working and didn't think of an idea that would be helpful for people until recently! See you there!

**Comment by source-c** [comment]
@exfrog gentle reminder :)
